perun-rpc/src/main/perl/removeHostsFromAllFacilities

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+use Perun::Agent;
+use Perun::Common qw(printMessage);
+
+sub help {
+	return qq{
+	Removes selected hosts (optional and destinations) from facilities. One or more hostnames is required field.
+	------------------------------------
+	Available options:
+	--hosts        | -H list Of Host names
+        --dest         | -d remove appropriated destinations
+	--batch        | -b batch
+	--help         | -h prints this help
+
+	};
+}
+
+my (@hosts, $dest, $batch);
+GetOptions ("help|h" => sub { print help(); exit 0;} ,
+	"batch|b" => \$batch,
+'hosts|H=s@{1,}' => \@hosts,
+"dest|d" => \$dest) || die help();
+
+# Check options
+unless (@hosts) { die "ERROR: hosts are required \n";}
+
+my $agent = Perun::Agent->new();
+my $facilitiesAgent = $agent->getFacilitiesAgent;
+my $servicesAgent = $agent->getServicesAgent;
+my $hostsAgent = $agent->getHostsAgent;
+
+# zruseni hosts
+while (@hosts) {
+    my $host=shift(@hosts);
+    my @hostobj=$facilitiesAgent->getHostsByHostname(hostname => $host);
+    unless (@hostobj) {printMessage "No host found for host $host", $batch; exit 0;} 
+    my @hostIds;
+    while (@hostobj) {
+        my $pom=shift(@hostobj);
+        my $hostId=$pom->getId;
+        push (@hostIds,$hostId);
+    } 
+    my @facilities = $facilitiesAgent->getFacilitiesByHostName(hostname => $host);
+    unless (@facilities) { printMessage "No facility found for host $host", $batch; exit 0;}
+    while (@facilities) {
+        my $fac=shift(@facilities);
+        my $facilityId=$fac->getId;
+        $facilitiesAgent->removeHosts(hosts => \@hostIds, facility => $facilityId);
+        printMessage("Host: $host successfully removed from the facility Id:$facilityId", $batch);
+    }
+    # zruseni destinations
+    if (defined $dest) {
+        @facilities = $facilitiesAgent->getFacilitiesByDestination(destination => $host);
+        unless (@facilities) { printMessage "No facility found for destination $host", $batch; exit 0;}
+        while (@facilities) {
+            my $fac=shift(@facilities);
+            my $facilityId=$fac->getId;
+            my @services = $servicesAgent->getAssignedServices(facility => $facilityId);
+            my $service;
+            my $typ="host";
```

Comment:
Are we sure, that ``$typ="host";`` will be always only "host" ( @licehammer )? We have also ``"user@host", "user@host:port", "service-specific"`` and other types. Does a tool caller have a knowledge of exact destination type ? I guess it should be either passed on input or based on destination retrieved from facility destinations by name and then deleted using original object.

Also please fix Czech variable name to English (including some comments in script). Thank you.